### PR TITLE
Updated background and padding

### LINKED ISSUE #19558: 🐛 Bug Report: [Docs] Permission Diagram Hard to Read
### 📜 Description

The permission digram under the "How Does It Work" heading is really hard to read: https://backstage.io/docs/permissions/overview#how-does-it-work

### 👍 Expected behavior

The digram should be easy to read

### 👎 Actual Behavior with Screenshots

image

### 👟 Reproduction steps

1. Go to: https://backstage.io/docs/permissions/overview#how-does-it-work

Note: the digram is hard to read

### 📃 Provide the context for the Bug.

Was just sharing these docs and noticed that this diagram is hard to read

### 🖥️ Your Environment

No really an environment issue

### 👀 Have you spent some time to check if this bug has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/backstage/backstage/blob/master/CODE_OF_CONDUCT.md)

### Are you willing to submit PR?

No, but I'm happy to collaborate on a PR with someone else

### PR DESCRIPTION
**Pull request by awanlin**
## Hey, I just made a Pull Request!

Updated the permissions digram to have a white background and roughly 10px padding all around

Closes #19558

|  Before | After  |  
|---|---|
|  Screenshot 2023-08-26 at 5 47 28 PM |  Screenshot 2023-08-26 at 5 47 17 PM |  

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: docs/assets/permissions/permission-framework-overview.drawio.svg
Comment (camilaibs) [inline docs/assets/permissions/permission-framework-overview.drawio.svg:1]: Would it be possible to add a white background via CSS to all images as an alternative when the dark theme is active?

**Review by camilaibs — APPROVED** [review]
Thanks for updating the image 🙌🏻 
Just left a question, nothing blocking this pull request 🙂

File: docs/assets/permissions/permission-framework-overview.drawio.svg
Comment (awanlin) [reply docs/assets/permissions/permission-framework-overview.drawio.svg:1]: Yeah, @vinzscam suggested that in the related issue. I went this route as I felt that getting it fixed quickly was better then having it be really hard to read. Maybe we could log a more general Issue with a help wanted label to add this?

Comment (camilaibs) [reply docs/assets/permissions/permission-framework-overview.drawio.svg:1]: Makes sense, I agree with you. Do you mind opening up the more general issue for us 🙏🏻 ?

Comment (awanlin) [reply docs/assets/permissions/permission-framework-overview.drawio.svg:1]: Sure, did a quick look and found this: https://github.com/backstage/backstage/issues/17712. Would that work or should I log something new for this?

**Review by freben — APPROVED** [review]
Merging in the mean time!

**Comment by awanlin** [comment]
Thanks @freben and @camilaibs 🚀

**Pull request merged**